### ISSUE TITLE
Receivers Index

### ISSUE THREAD

**Issue by colleenmcguckin** [colleenmcguckin/UH#16]
Form:
- [x] Show more relevant details
- [x] make links work( go to show page)
- [x] Show map

Function:
- [x] Sort / Filter bullshit
- [x] Paginate

**Comment by colleenmcguckin** [comment]
everything is done except auto filtering any receiver who has restrictions that would disallow the donation
The filter should function as follows:
- get rid of receivers who have restrictions that are included in the donation
- this will be either a category through the food, or a storage_temp on the food, which is one of many items on the donation
- Receiver where each receiver.restrictions.first.categories != each donation.items.(first/2nd/etc).food.category && each restricions.first.storage_temps != each donation.items.(first/2nd/etc).food.storage_temp

```
def self.filter donation
  categories_to_exclude = donation.items.map(&:food).map(&:category)
  storage_temps_to_exclude = donation.items.map(&:food).map(&:storage_temp)

  Receiver.includes(restrictions: [:categories, :storage_temps])...???
end
```

WIP is on the receivers_index branch

**Comment by colleenmcguckin** [comment]
closed by #56
